### ISSUE TITLE
Add Deploy section to docs

### ISSUE THREAD

**Issue by arogachev** [yiisoft/yii2#9721]
It will be useful if this section will be presented in the docs with some solutions and tips.

I added Yii2 recipes for [Deployer](https://github.com/deployphp/deployer) for both basic and advanced application templates 

Maybe include this too?

We can extend it more of course.

**Comment by dynasource** [comment]
@arogachev, adding a deploment section is indeed useful. Can you write a short manual how to implement this for 'Deployer'?. It would not have to be large, 1 section per deploy tool would be enough.

**Comment by samdark** [comment]
There are many deployment tools and many different variants of deployments. I guess that if even most used tools are covered, the guide could be a size of a small book. Anyway, I'm all for having it.

So... if anyone has a generic-enough solution to deployment with certain tool, post a guide here in comments. We'll see what we would be able to do with that info.

**Comment by dynasource** [comment]
agreed. We should keep the docs lean & mean. Deploying is an important aspect so it deserves some attention. I will create a first version for it.
